### PR TITLE
make selected tabs standout

### PR DESCRIPTION
**Pull request by rycastaneda**
Remove duplicated section name

Difference
Live 
image

Changes
image

### CODE REVIEW

**Comment by ravenjohn** [comment]
<a href="http://4380e460.ngrok.com/anyTV/music-dashboard/pull/380" target="_blank">ReviewNinja

File: app/styles/front/components/_tabs.scss [hunk 1]
```diff
@@ -8,12 +8,15 @@
...
+                font-size: larger;
+                letter-spacing: 2.8px;
                 a {
                     color: $darker-grey-color;
                     &.active,
                     &:hover {
                         border-bottom: 3px solid $dark-orange-color;
-                        font-weight: bold;
+                        font-weight: bolder;
+                        // font-size: 15px;
```
Comment (ToastedIceCream) [inline app/styles/front/components/_tabs.scss]: remove dead code / not used code?

Comment (rycastaneda) [inline app/styles/front/components/_tabs.scss]: updated

**Comment by sibayanjasper** [comment]
:AllenApproves:

**Pull request merged**